docs/excel/custom-functions-overview.md

```diff
@@ -1,8 +1,8 @@
-# Create custom functions in Excel (Preview)
+ (# Create custom functions in Excel (Preview)
```

Comment:
Looks like an open parenthesis was inadvertantly added to the first line...can you please remove it? Thanks!